pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -139,6 +139,7 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 		cidr.IP.To4()[3] += 1
 
 		json := fmt.Sprintf(NET_CONFIG_TEMPLATE, BridgeName, plugin.MTU, network.DefaultInterfaceName, podCIDR, cidr.IP.String())
+		glog.V(2).Infof("CNI network config set to %v", json)
```

Comment:
Hah. Suggest either renaming the function, or allowing generic events. And logging when a !POD_CIDR_CHANGE is observed at Warningf.